backend/internal/service/handler/review.go

```diff
@@ -0,0 +1,104 @@
+package handler
+
+import (
+	"fmt"
+	"platnm/internal/storage"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type ReviewHandler struct {
+	reviewRepository storage.ReviewRepository
+}
+
+func NewReviewHandler(reviewRepository storage.ReviewRepository) *ReviewHandler {
+	return &ReviewHandler{
+		reviewRepository,
+	}
+}
+
+func (h *ReviewHandler) GetReviews(c *fiber.Ctx) error {
+	review, err := h.reviewRepository.GetReviews(c.Context())
+	if err != nil {
+		return err
+	}
+
+	return c.Status(fiber.StatusOK).JSON(review)
+}
+
+func (h *ReviewHandler) GetReviewById(c *fiber.Ctx, mediaType string) error {
+	id := c.Params("id")
+	offsetstr := c.Query("offset", "0")
+	limitstr := c.Query("limit", "10")
+	//limitParam := r.URL.Query().Get("limit")
+	//offsetParam := r.URL.Query().Get("offset")
+	//review, err := h.reviewRepository.GetReviewsByID(id, mediaType, c.Context())
+
+	var offset int
```

Comment:
maybe move offset limit stuff to helper func to clean up handler func. or create utils package with this functionality since we may need in the future